iot/iot-tenant-cleaner/src/main/java/io/enmasse/iot/tools/cleanup/Application.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.iot.tools.cleanup;
+
+import io.vertx.core.Future;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Application {
+
+    private static final Logger log = LoggerFactory.getLogger(Application.class);
+
+    public static void main(String args[]) {
+
+        if (args.length < 1){
```

Comment:
Do we require a config file? I though that we can also use env-vars, which would be much simpler in the Kubernetes environment. Otherwise we could need to manage a ConfigMap for each Job that we create.

Reply:
env-vars are fine as well, I'll edit !